Src/AutoFixture/Kernel/PropertySpecification.cs

```diff
@@ -95,7 +95,9 @@ private bool PropertyIsCompatibleWithTargetType(object request)
 
         private bool PropertyHasTargetName(object request)
         {
-            return ((PropertyInfo)request).Name == this.targetName;
+            return ((PropertyInfo)request)
+                   .Name
+                   .Equals(this.targetName, StringComparison.OrdinalIgnoreCase);
```

Comment:
Isn't this a breaking change?

What if someone wishes to use this class, but with a case-sensitive string comparison?